### PR TITLE
feat(vxASTRO): updates

### PR DESCRIPTION
**Pull request by maksk7**

### CODE REVIEW

**Review by ShadoySV — APPROVED** [review]

**Review by P-Yevhenii — APPROVED** [review]

File: contracts/voting_escrow/src/contract.rs [hunk 1]
```diff
@@ -786,14 +792,36 @@ fn withdraw_early_callback(
...
     if !return_astro_amount.is_zero() {
-        let transfer_msg = SubMsg::new(WasmMsg::Execute {
-            contract_addr: config.astro_addr.to_string(),
-            msg: to_binary(&Cw20ExecuteMsg::Transfer {
-                recipient: slashed_funds_receiver.to_string(),
-                amount: return_astro_amount,
-            })?,
-            funds: vec![],
-        });
+        // Check that slashed_funds_receiver is a escrow_fee_distributor contract
```
Comment (epanchee) [inline contracts/voting_escrow/src/contract.rs:795]: an escrow...

**Review by epanchee — APPROVED** [review]

File: contracts/voting_escrow/src/contract.rs (hunk 1 above)
Comment (maksk7) [reply contracts/voting_escrow/src/contract.rs]: Thanks, fixed

File: contracts/voting_escrow/src/contract.rs [hunk 2]
```diff
@@ -786,14 +792,36 @@ fn withdraw_early_callback(
...
+        // Check that slashed_funds_receiver is an escrow_fee_distributor contract
+        // otherwise transfer return_astro_amount to slashed_funds_receiver address
+        let version: StdResult<ContractVersion> =
+            deps.querier.query(&QueryRequest::Wasm(WasmQuery::Raw {
+                contract_addr: slashed_funds_receiver.to_string(),
+                key: CONTRACT.as_slice().into(),
+            }));
+        let transfer_msg = match version {
+            Ok(ContractVersion { contract, .. })
+                if contract.eq("astroport-escrow_fee_distributor") =>
```
Comment (ShadoySV) [inline contracts/voting_escrow/src/contract.rs:804]: There are only dashes, no underscores in the contract name

Comment (maksk7) [reply contracts/voting_escrow/src/contract.rs]: Changed underscores to dashes in fee distributor CONTRACT_NAME, thanks

**Pull request merged**